portability-gateway/src/main/java/org/dataportabilityproject/gateway/crypto/AesSymmetricKeyGenerator.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
This and the other files look like they are dups of the ones in #134 - They should be cleaned up when you merge master in right?